cluster/userdata-master.yaml

```diff
@@ -756,6 +760,36 @@ write_files:
               emptyDir:
                 medium: Memory # share a tmpfs between the two containers
 
+  - path: /srv/kubernetes/manifests/mate-de.yaml
+    content: |
+      apiVersion: extensions/v1beta1
+      kind: Deployment
+      metadata:
+        name: mate
+        namespace: kube-system
+      spec:
+        replicas: 1
+        template:
+          metadata:
+            labels:
+              app: mate
+          spec:
+            containers:
+            - name: mate
+              image: pierone.stups.zalan.do/teapot/mate:v0.0.4-32-g0af5e78
```

Comment:
I think mate should be in the OSS registry (and we should open source) --- having Pier One here depends on correct credentials